fpdf/fpdf.py

```diff
@@ -2327,7 +2328,7 @@ def _render_styled_cell_text(
         elif new_x == XPos.WCONT:
             self.x = s_start + s_width - self.c_margin
         elif new_x == XPos.CENTER:
-            self.x = (s_start + s_start + s_width) / 2.0
+            self.x = s_start + s_width / 2.0
```

Comment:
I don't think that change is correct.
We need the center of the actual text. `s_start` is the left end, `s_start + s_width` is the right end, and both together divided by 2 are the center. Your version is `s_start / 2 ` too far to the left.

Reply:
Yes: https://docs.python.org/3/reference/expressions.html#operator-precedence
😉